quantum/quantum.h

```diff
@@ -254,7 +244,7 @@ void tap_code16(uint16_t code);
 void backlight_init_ports(void);
 void backlight_task(void);
 
-#ifdef BACKLIGHT_BREATHING
+    #ifdef BACKLIGHT_BREATHING
```

Comment:
```suggestion
#ifdef BACKLIGHT_BREATHING
```
preprocessor blocks should stay left aligned imo (opinion)